.github/workflows/android-build.yml

```diff
@@ -0,0 +1,41 @@
+name: Android
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: NDK-C++${{matrix.std}}-${{matrix.abi}}-${{matrix.build_type}}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+          std: [98, 11, 14, 17, 20]
+          abi: [arm64-v8a, armeabi-v7a, x86_64, x86]
+          build_type: [Debug, Release, RelWithDebInfo]
```

Comment:
Is `RelWithDebInfo` actually necessary? I'd say `Debug` and maybe `Release` are sufficient.